src/Header.js

```diff
@@ -0,0 +1,65 @@
+import { Nav, Navbar, NavDropdown } from 'react-bootstrap'
+import Toolbar from './Toolbar'
+
+import { useMemo } from 'react'
+
+const select = (state, setState, setModules, callback) => e => {
+  const val = e.target.text
+  if (val !== state) {
+    setModules([])
+    setState(val)
+    if (callback) callback(val)
+  }
+}
+
+const Header = ({ API, timetableState}) => {
+  let {sessions, year, setYear, session, setSession, setSelectedModules, darkMode} = timetableState
+  const selectYear = select(year, setYear, setSelectedModules, yr => {
+    const s = sessions[yr]
+    setSession(s?.[s.length-1] || '')
+  })
+  const selectSession = select(session, setSession, setSelectedModules)
+  const years = useMemo(() => Object.keys(sessions).reverse(), [sessions])
+  const mode = darkMode ? 'dark' : 'light'
+
+  return <Navbar variant={mode} bg={mode} expand="md" sticky="top">
+    <Navbar.Brand href="https://cssa.club/">
+      <img
+        alt="CSSA logo"
+        src="/cssa-mono.svg"
+        width="30"
+        height="30"
+        style={darkMode ? {
+          filter: 'invert(.75)'
+        } : {
+          filter: 'invert(.25)'
+        }}
+        className="d-inline-block align-top mr-1"
+      />{' '}
+      ANU Timetable
```

Comment:
Sound good, I really wasn't sure about that behaviour. I'll try and implement it this evening

Reply:
I've moved the CSSA link to the logo, but the state-clearing functionality would be tricky now that state is restored from local storage. So I've removed it - might be a better experience, in case a user accidentally clicked the header while on mobile